x-pack/plugins/triggers_actions_ui/public/application/sections/alert_summary_widget/alert_summary_widget.tsx

```diff
@@ -22,6 +22,7 @@ export const AlertSummaryWidget = ({
   fullSize,
   onClick = () => {},
   timeRange,
+  shouldHideCharts,
```

Comment:
Thank you for the update! This one is missed.

Reply:
Fixed forgot to commit it, thanks :)